src/System.Private.CoreLib/shared/System/Collections/Generic/IDictionary.cs

```diff
@@ -10,7 +11,7 @@ namespace System.Collections.Generic
     // Keys can be any non-null object.  Values can be any object.
     // You can look up a value in an IDictionary via the default indexed
     // property, Items.
-    public interface IDictionary<TKey, TValue> : ICollection<KeyValuePair<TKey, TValue>>
+    public interface IDictionary<TKey, TValue> : ICollection<KeyValuePair<TKey, TValue>> // TODO-NULLABLE-GENERIC: TKey should be non-nullable
```

Comment:
Please add a `// TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761` comment on TryGetValue; its out argument may be null even if TValue is non-nullable.